sorting/recursive_bubble_sort.cpp

```diff
@@ -0,0 +1,165 @@
+/**
+ * @file recursive_bubble_sort.cpp
+ * @author Aditya Prakash (https://adityaprakash.tech)
+ * @brief This is an implementation of recursive version of Bubble sort
+algorithm
+ *
+ * @details
+ * The working principle of the Bubble sort algorithm:
+
+Bubble sort is a simple sorting algorithm used to rearrange a set of elements in
+ascending or descending order. Bubble sort gets its name from the fact that data
+"bubbles" to the top of the dataset.
+
+ * What is Swap?
+
+To swap two numbers mean that we interchange their values.
```

Comment:
All text should have a `* ` at the beginning.

Reply:
Fixed in latest commit :)